hawkular-inventory-rest-api/src/main/webapp/WEB-INF/jboss-web.xml

```diff
@@ -19,5 +19,8 @@
 -->
 <jboss-web>
   <context-root>/hawkular/inventory</context-root>
-  <security-domain>keycloak</security-domain>
+
+  <!-- comment the following line out to disable keycloak auth -->
+  <!--<security-domain>keycloak</security-domain>-->
```

Comment:
Should this really be left commented in the default state? I think we should by default function authz'ed.